cmd/frontend/db/external_services.go

```diff
@@ -81,6 +80,36 @@ func (c *externalServices) Update(ctx context.Context, id int64, update *Externa
 	})
 }
 
+type externalServiceNotFound struct {
```

Comment:
```suggestion
type externalServiceNotFoundErr struct {
```

It's customary to suffix error types of `Err` in our codebase (e.g., https://sourcegraph.sgdev.org/github.com/sourcegraph/sourcegraph@574ba50/-/blob/cmd/frontend/db/users.go#L32:4)

Reply:
These should all be suffixed with `Error`: https://github.com/golang/go/wiki/Errors#naming